### DISCUSSION TITLE
Source code to demo ?

### DISCUSSION THREAD

**Discussion by nlindblo** [shannonhochkins/ha-component-kit#67] [General]
Hi,

Where can I find the source code to the demo after you click overview->default in the documentation ? I'd like to figure out how the side bar is done (and I am not a developer)

Many thanks

**Comment by shannonhochkins** [comment]
Here you go!

https://github.com/shannonhochkins/ha-component-kit/blob/master/stories/0.overview.stories.tsx

If you get stuck, give me a shout, happy to help :)

**Reply by shannonhochkins** [reply]
Also, I've started working on an addon for home assistant for users like yourself that don't have full coding experience, [home assistant addon](https://github.com/shannonhochkins/hakit) keep an eye on it if you're interested :)
